lib/actions/frontend/start.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { fork } = require('child_process')
+const { join } = require('path')
+const AppSettings = require('../../app/AppSettings')
+
+let instance
+
+class StartAction {
+  static getInstance () {
```

Comment:
action dont need to be a singleton since bin/sgcloud will only create one instance anyway